webcompat/templates/contact.html

```diff
@@ -27,17 +27,17 @@ <h1 class="headline-1 highlight">Say hello!</h1>
 		<div class="grid-cell x1">
 			<h2 class="headline-2">Channels</h2>
 		    <ul>
-		    	<li>Join us 
+		    	<li>Join us
 		    		<a href="https://wiki.mozilla.org/IRC#Connect_to_the_Mozilla_IRC_server" title="Join us on IRC">
 		    			on IRC
 		    		</a>
-		    	</li> 
+		    	</li>
 		    	<li>
-		    		Join us on our 
+		    		Join us on our
 		    		<a href="https://lists.mozilla.org/listinfo/compatibility" title="Join us on our mailing list">
 		    			mailing list
 		    		</a>
-		    	</li> 
+		    	</li>
 		    	<li>
 		    		Join our
 		    		<a href="https://webcompat.slack.com/" title="Join our Slack">
```

Comment:
I know #2083 is not resolved yet, so we should resolve it one way or the other before we ship this refactor.

Reply:
Sounds good. :)

Reply:
Let's merge and I'll put #2083 on the v1 list?